runtime/src/main/java/org/corfudb/runtime/ViewsMaintenance.java

```diff
@@ -23,39 +24,71 @@
 
 @Slf4j
 @NotThreadSafe
-public class ViewsGarbageCollector {
+public class ViewsMaintenance {
 
 
     private long trimMark = Address.NON_ADDRESS;
 
     @Getter
     private boolean started;
 
-    final ScheduledExecutorService gcThread = Executors.newSingleThreadScheduledExecutor(
+    @Getter
+    @Setter
+    long objectSyncDelayInSeconds = 30;
```

Comment:
One purpose of periodical sync is to avoid expensive syncing due to running out of cache. 
Is it better if objectSyncDelayInSeconds is a function (probably just as simple as a linear function) of the runtime cache size?

Reply:
Its not just a function of the cache, but also the time it takes to resolve the backpointers. I also want to keep the parameters simple.